### DISCUSSION TITLE
Runtime error - loop is closed.

### DISCUSSION THREAD

**Discussion by idan-dagan-imagenai** [encode/starlette#2331] [Q&A] [answered]
![image](https://github.com/encode/starlette/assets/117266213/7b32df86-52fc-40a7-966f-bba63ccd8bdb)

This is happening when I am exiting from my pod - I have two event loops running - one inside my main thread and one inside another thread. I am trying understand why this is happening in my kubernetes cluster but not locally.. what am I missing?

**Comment by Kludex** [comment, marked as answer]
I don't know. More information is needed.
